src/StateMangement/OKRReducers.ts

```diff
@@ -49,7 +52,7 @@ export const reducer = (state: OKRMainState = initialState, action) => {
         });
         // If an area isn't already selected, set the first areas as the selected area
         draft.selectedArea = draft.selectedArea ? draft.selectedArea : draft.areas[0];
-        draft.selectedArea = draft.selectedArea ? draft.selectedArea : draft.areas[0];
+        draft.identityProvider = new IdentityProvider();
```

Comment:
Did you do this for any perf gain, or just for a cleaner implementation?

Reply:
mainly for the cleaner implementation, we do not need to initialize the same provider 3 times.